assets/js/main.js

```diff
@@ -0,0 +1,29 @@
+
+//Main functions here
+const key = '1f54bd990f1cdfb230adb312546d765d';
+let loading = false;
+let result;
```

Comment:
Why use a global variable to store the results instead of passing it to the request callback?

Reply:
You're right its better just passing it in the request callback.